docs/source/gpu.rst

```diff
@@ -25,29 +49,44 @@ As a worked example, you may want to view this talk:
                 type="video/mp4">
     </video>
 
-High Level Collections
-----------------------
+`Dask and V100s for Fast, Distributed Batch Scoring of Computer Vision Workloads <https://developer.nvidia.com/gtc/2019/video/s9198>`_
+
+Having parallel tasks use GPUs
+------------------------------
+
+Tasks like running experiments of different parameters of a machine learning model on a GPU
+can be extremely parallelized. Dask can be used to run each experiment trial concurrently,
+dramatically speeding up the process. Parallelizing experiment runs can be useful with or without
+GPUs, however because models that are trained on GPUs often take an especially long time to train
+there is a lot of benefit in this situation especially.
```

Comment:
Is this why? I thought there was something special about GPUs that made them particularly well-suited for parallel computing.

Reply:
We talked about this in a zoom call, but I think the answer is "sort of", Like GPUs can run many concurrent computations (which is how they do video games graphics quickly), but given a task that runs on a single GPU and then running the same task concurrently with multiple GPUs is no better than having a single task on a CPU that is then run multiple tasks currently on multiple CPUs. I'll try and make this clearer